### DISCUSSION TITLE
ScriptReference/Rigidbody2D

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#601] [Notes]
# ScriptReference/Rigidbody2D

https://docs.unity3d.com/ScriptReference/Rigidbody2D.html

**Comment by coolblueflame** [comment]
If you manually set a Rigidbody2D's position or rotation using the respective properties or SetRotation, and then you call MovePosition or MoveRotation before a physics step is run, the move instructions will be ignored! If you need this functionality to work, run Physics2D.Simulate(0f) before calling the move methods.

Untested, but it's likely the standard Rigidbody has the same behaviour.
